src/insns/sentry_32bit.adoc

```diff
@@ -22,6 +22,11 @@ Description::
 Capability `cd` is written with the capability in `cs1` with its type bit set to 1.
 Attempting to seal an already sealed capability will lead to the tag of `cd` being set to 0.
 
+NOTE: The SENTRY instruction may give rise to an illegal instruction fault when
```

Comment:
Maybe we could change this to "when the implementation does not support capability type 1 (unrestricted sentry)", since you could still imagine an implementation that has seal/unseal permissions but allows unrestricted sentry (e.g. CHERIv9).

Reply:
Quite

Reply:
Done